### ISSUE TITLE
Design Call #22

### ISSUE THREAD

**Issue by Zaxounette** [btcpayserver/organization#66]
```meta
Date: 2021-11-04
Time: 5:00pm
Timezone: UTC
Duration: 1h
UTCTime: 2021-11-04 17:00 UTC
```

### Agenda

Comment below to add items to the agenda.

### Check your timezone

https://everytimezone.com/s/2c41d084

### Join the call

https://meet.jit.si/DesignCallsROCK

### Calendar invite

Subscribe to the BTCPay Server calendar. More info [here](https://github.com/btcpayserver/organization#calendar).

**Comment by dennisreimann** [comment]
* Wallet/Lightning Settings modification button/link/whatever
* Wallet Settings merge (btcpayserver/btcpayserver#2626 and btcpayserver/btcpayserver#2206)

**Comment by dstrukt** [comment]
^^ I'm hoping some of my latest mocks will help address both of those issues.

Proposal:
- Address @dennisreimann's list
- Store Centric UI (as it pertains to the above, generally speaking, and a series of specific questions to define and unblock future work - will post an up-to-date list of items closer to the call)

Would imagine that'll take the entire time, but have consolidated enough of this for likely a single call, and each subsequent call will just be focused views and existing PRs (as usual), and won't take up the entire call for the future.

**Comment by dstrukt** [comment]
Questions for reference. 
They will make more sense after we step through prototypes.

<img width="621" alt="Screen Shot 2021-11-04 at 1 04 29 AM" src="https://user-images.githubusercontent.com/6250771/140278404-d7f2c3b8-9687-4a96-9030-58cae97e7c5e.png">

**Comment by pavlenex** [comment]
I'd like to add to the agenda that we finally need to come up with a concept for  a store-centric UI.

**Comment by Zaxounette** [comment]
**Design Call N°22:**
**Attendees:** @Webworthy, @Zaxounette, @pavlenex, Satwo, @dstrukt 

----

Dstrukt shows us the Figma mirror versions of his mocks for the Store centric view flow.

The first version presented contains a horizontal sub-navigation menu nested underneath the Store selection dropdown.

The second version relates to the Left Navigation bar containing all the nav items including the server settings.
The third version contains all Left navigation items except account, server settings and server wide invoices which are nested on the top nav bar.

The first version is clean and clear, but the drawback is one-click accessibility of advanced features that are nested under the horizontal nav bar and then a left nav bar in certain instances. It is potentially too first-time-user friendly and over-simplifies potentially too much.

The second and third versions are more complex, potentially more cluttered, but enables one-click access to advanced features for more experienced users.

The full left side-bar concept has the preference of the majority.
In regards to that view, with multiple apps/plugins as well as wallet and wallet types, the Left side bar could become quite long/large.
* In this case the bottom left items revolving around server settings and account settings would need to be fixed in some way
  * We discuss solutions regarding the bottom left nav items in different ways (bottom left fixed VS moving them top left under the BTCPay logo as icons only)

We discuss again the different versions and go back and forth on the tradeoffs of the different versions.

Dashboard:
Do we want a global dashboard? Yes, to be able to provide information on multiple stores at a time.
* Idea of potentially using the BTCPay logo as a link to the global dashboard
* Or tucking another dashboard nav item under the logo

Dstruck to iterate based on feedback and new ideas.
